mmv1/products/bigquery/Job.yaml

```diff
@@ -62,6 +62,7 @@ examples:
     primary_resource_id: 'job'
     vars:
       job_id: 'job_query_continuous'
+      project_id: 'project_query_continuous'
```

Comment:
is it necessary to be explicit with the project that we're using for testing?

We get the following error when attempting to use this `project_id`

```hcl
      Error: Error creating Job: googleapi: Error 400: Invalid IAM service account 'bigquery-runner@tf_test_project_query_continuousrd3jjo12eo.iam.gserviceaccount.com'. Service account should be in service_account_id@project-id.iam.gserviceaccount.com format. SERVICE_ACCOUNT_ID must be between 6 and 30 characters, and can contain lowercase alphanumeric characters and dashes. PROJECT_ID must contain 6-63 lowercase letters, digits, or dashes., invalid
```

Reply:
No, it will probably be better to use `service_account_email` var instead. Updated